db/seeds.rb

```diff
@@ -1,7 +1,52 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "Clearing out the battlefield 🌋..."
+User.destroy_all
+PlayerGroup.destroy_all
+Group.destroy_all
+Preference.destroy_all
+Rulebook.destroy_all
+Genre.destroy_all
+puts "Battlefield Cleared ⛰"
+
+puts "Summoning Gandalf 🧙🏻‍♂️"
+gandalf = User.new
+gandalf.email = 'gandalf@gmail.com'
+gandalf.nickname = 'GandalfTheWhite'
+gandalf.password = '123456'
+gandalf.city = 'Tokyo'
+gandalf.save
+puts "Gandalf Summoned: 'You Shall Not Pass! 🧖🏼‍♂️'"
+
+puts "Generating NPCs..."
+40.times do
+  user = User.new
+  user.email = Faker::Internet.email
+  user.nickname = Faker::Internet.username
+  user.password = '123456'
+  user.city = Faker::Address.city
+  user.save
+  puts "#{user.nickname} from #{user.city} created"
+end
+puts "All NPCs Generated"
+
+puts "Generating Groups..."
+50.times do
```

Comment:
too many groups
you usually want 4-5 players per group so create just 20 of them